egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -42,6 +43,18 @@ def GetArgs():
     num_target_group.add_argument("--ali-dir", type=str,
                                   help="alignment directory, from which we derive the num-targets")
 
+    # CNN options
+    parser.add_argument('--cnn-layer', type=str, action='append',
+                        help="CNN parameters at each CNN layer, e.g. --filt-x-dim=3 --filt-y-dim=8 "
+                        "--filt-x-step=1 --filt-y-step=1 --num-filters=256 --pool-x-size=1 --pool-y-size=3 "
+                        "--pool-z-size=1 --pool-x-step=1 --pool-y-step=3 --pool-z-step=1", default = None)
+    parser.add_argument("--cnn-bottleneck-dim", type=int,
+                        help="Output dimension of the linear layer at the CNN output "
+                        "for dimension reduction, e.g. 256", default=0)
+    parser.add_argument("--cepstral-lifter", type=float,
+                        help="Here we need the scaling factor on cepstra in the production of MFCC"
+                        "to cancel out the effect of lifter, e.g. 22.0", default=22.0)
+
     # General neural network options
     parser.add_argument("--splice-indexes", type=str, required = True,
```

Comment:
Explain how adding the CNN layers affects interpretation of the splice-indexes options in the help string.